### PR TITLE
Update websiteaccount_card.php

### PR DESCRIPTION
**Pull request by dolibit-ut**

### CODE REVIEW

File: htdocs/website/websiteaccount_card.php [hunk 1]
```diff
@@ -43,7 +43,7 @@
 $object = new SocieteAccount($db);
 $extrafields = new ExtraFields($db);
 $diroutputmassaction = $conf->website->dir_output.'/temp/massgeneration/'.$user->id;
-$hookmanager->initHooks(array('websiteaccountcard')); // Note that conf->hooks_modules contains array
+$hookmanager->initHooks(array('websiteaccountcard'));   //  Note that conf->hooks_modules contains array
```
Comment (eldy) [inline htdocs/website/websiteaccount_card.php:46]: Note that adding a space is useless because tool to sanitize and standardize code and a lot of IDE remove them to keep only 1 space. So this change will probably be reverted by another PR done by someone else.

File: htdocs/website/websiteaccount_card.php [hunk 2]
```diff
@@ -32,9 +32,9 @@
 $langs->loadLangs(array("website", "other"));
 
 // Get parameters
-$id = GETPOST('id', 'int');
+$id         = GETPOST('id', 'int');
```
Comment (eldy) [inline htdocs/website/websiteaccount_card.php:35]: Note that adding a space is useless because tool to sanitize and standardize code and a lot of IDE remove them to keep only 1 space. So this change will probably be reverted by another PR done by someone else.

Comment (dolibit-ut) [reply htdocs/website/websiteaccount_card.php:35]: unterstood
but that is a pity, because it allows better & faster recognition of patterns and errors
